src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.component.ts

```
@@ -1,34 +1,158 @@
+    readonly faSortDown = faSortDown; // Added for sorting icons
+    readonly faFilter = faFilter;
+    readonly faUpRightAndDownLeftFromCenter = faUpRightAndDownLeftFromCenter;
+    readonly SortingOrder = SortingOrder;
+    readonly MAX_FEEDBACK_DETAIL_TEXT_LENGTH = 150;
+
+    // Added sortIcon computed function to dynamically set the sort icon
+    readonly sortIcon = computed(() => (this.sortingOrder() === SortingOrder.ASCENDING ? this.faSortUp : this.faSortDown));
+
+    private localStorage = inject(LocalStorageService);
+    selectedFiltersCount = 0;
+    private hasAppliedFilters: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)